spring-graphql-api/src/main/java/com/okta/developer/demo/SpringBootApiConfig.java

```diff
@@ -6,15 +6,25 @@
 import org.springframework.data.neo4j.core.ReactiveDatabaseSelectionProvider;
 import org.springframework.data.neo4j.core.transaction.ReactiveNeo4jTransactionManager;
 import org.springframework.data.neo4j.repository.config.ReactiveNeo4jRepositoryConfigurationExtension;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.web.SecurityFilterChain;
 import org.springframework.transaction.ReactiveTransactionManager;
 
+import static org.springframework.security.config.Customizer.withDefaults;
+
 @Configuration
-public class SpringBootGraphQlApiConfig {
+public class SpringBootApiConfig {
```

Comment:
Did this class get renamed in the blog post as well?

Reply:
Yes